app/models/news_article.rb

```diff
@@ -4,9 +4,11 @@ class NewsArticle < Newsesque
   include ::Attachable
   include Edition::AlternativeFormatProvider
   include Edition::CanApplyToLocalGovernmentThroughRelatedPolicies
+  include Edition::WorldwideOrganisations
 
   validates :news_article_type_id, presence: true
   validate :non_english_primary_locale_only_for_world_news_story
+  validate :worldwide_organisation_association_only_for_world_news_story
```

Comment:
I think we can refactor this to clean it up quite a bit. If we had:
```
def world_news_story?
...etc
```
Then we can use the normal `validates:` with `presence: true` if that method ^ returns true and `absence: true` if that method returns false. The method on 67 would also just return `!!world_news_story?`.